frontend/src/components/explorer/gemBrowser/GemSearch.vue

```diff
@@ -1,6 +1,17 @@
 <template>
   <div v-if="model" id="gem-search-wrapper">
     <div class="field has-addons m-0">
+      <a
+        href="/search"
+        id="globalSearchButton"
+        class="button is-rounded is-outlined is-success"
+        title="Global GEM search"
+      >
```

Comment:
Surely we should use a `routerLink` instead of an ordinary `a` element to avoid full page reload?